### ISSUE TITLE
It's not possible to change the text color of the tooltip on TreeMap diagram

### ISSUE THREAD

**Issue by dmsmaxim** [amcharts/amcharts4#516]
Example: 
https://codepen.io/dmsmaxim/pen/oQvNjK

I'm doing this:
```
level1.tooltip.getFillFromObject = false;
level1.tooltip.background.fill = am4core.color("white");
level1.tooltip.label.fill = am4core.color("red");
```
but the text is still white

**Comment by notacouch** [comment]
You can also use [Visual Formatting](https://www.amcharts.com/docs/v4/concepts/formatters/formatting-strings/#Visual_formatting):

Replace:
```javascript
level1.tooltip.label.fill = am4core.color("red");
```

with:
```javascript
level1_column.tooltipText ="[#f00]{parentName} {name}: {value}";
```
Screenshot:
![image](https://user-images.githubusercontent.com/2334143/47875094-045ca180-ddec-11e8-99bf-1bfa743c5c85.png)

Does that work for you?

**Comment by dmsmaxim** [comment]
Thanks for the tip.
Yes, I can use it. But formatting has some limitations.
`level1.tooltip.label.fill = am4core.color("red");` should work anyway...

**Comment by martynasma** [comment]
Let me jump in.

I agree that `label.fill` should work IF [`autoTextColor`](https://www.amcharts.com/docs/v4/reference/tooltip/#autoTextColor_property) is set to `false`.

Except it doesn't, which I'd consider a bug.

We are going to look into this if there's a better workaround, or we can make a fix.

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.73](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.73).

## [4.0.0-beta.73] - 2018-11-07

### Changed
- `axisFills.template.interactionsEnabled` is now set to `false` by default. If you need to attach events on an axis fill, you will need to set it to `true`.

### Added
- `autoDispose` property added to `Series` and `Axis` classes with default value `true`. This means that series/axis will be automatically disposed when removed from chart's `series`, `xAxes`, or `yAxes` lists.
- It is now possible to have 3D stacked columns (by setting `clustered = false` on series).

### Fixed
- If chart container was moved in DOM tree, it stopped sensing resize and was not updating its size anymore.
- Sometimes `ValueAxis` could show zero label with a minus sign.
- Arrows on `MapLine` were not drawn.
- It was not possible to change the text color of the tooltip on `TreeMap` diagram.
- `axis.tooltipPosition = "pointer"` was not working as it should - always follow the mouse/pointer.
- If series had `data` set directly, it was still using chart's global `data`, resulting in combined datasets and potentially visual anomalies.
- Setting `cellStartLocation` did not have any effect on an already inited chart.
- Line series could be cut off in the beginning (when zoomed-in closely).
- Vertical axis ticks were not being positioned properly if axis had a title.
- 3D columns were not using chart's `depth`/`angle`.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
